backend/src/users/users.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Body, Controller, Get, Post } from '@nestjs/common';
+import { CreateUserDto } from './users.dto';
+import { UsersService } from './users.service';
+
+@Controller('users')
+export class UsersController {
+  constructor(private usersService: UsersService) {}
+
+  @Post()
+  create(@Body() createUserDto: CreateUserDto) {
```

Comment:
I see some warning from backend pipeline here. Can't we use ValidationPipe you've created?

Reply:
The warning suggests using validation pipes individually for each endpoint. For now, it is used globally and I believe Linter doesnt see this.